src/sympc/approximations/sigmoid.py

```diff
@@ -41,3 +44,31 @@ def sigmoid(tensor: MPCTensor, method: str = "exp") -> "MPCTensor":
             result += (tensor ** d) * weights[i + 1]
 
         return result
+
+    elif method == "chebyshev":
+        # Reference: http://www.nnw.cz/doi/2012/NNW.2012.22.023.pdf
+        # Make sure the elements are all positive
+        _sign = sign(tensor)
+        positive_tensor_rescaled = tensor * _sign / 8
+        p = 8
+        q = 8
+        scaler = ((1 + positive_tensor_rescaled) / 2) ** (q + 1)
+
+        def factorial(n):
```

Comment:
MAYBE: Cache?

Reply:
Cache the factorial? Considering integers, the computation time should be negligible already.